.gitignore

```diff
@@ -8,7 +8,7 @@ out.json
 scripts/def.json
 dist
 readme.html
-
+.vscode/
```

Comment:
The `.vscode` directory is important for project-level settings that we'd like to recommend everyone use (we currently don't do it). It shouldn't be ignored. 
```suggestion

```